backends/aerospike.go

```diff
@@ -13,59 +15,111 @@ import (
 const setName = "uuid"
 const binValue = "value"
 
-type Aerospike struct {
+type AerospikeDB interface {
```

Comment:
I don't think we need an `AerospikeDB` interface here. Do you expect this interface to be implemented by anything else other than a struct containing an Aerospike client?

Reply:
Its main use is to be able to add test coverage to functions `Get(ctx context.Context, key string) (string, error)` and `Put(ctx context.Context, key string, value string, ttlSeconds int) error`.  Before these additions, the two functions didn't have much test coverage because it was hard to mock the `Aerospike` client error scenarios. Therefore, I thought an interface was the way to go, because `AerospikeDB` was implemented as `errorProneAerospikeClient` and `goodAerospikeClient` inside `backends/aerospike_test.go` and 100% coverage was reached.

What to do you think? I'm open to suggestions on this front.

Reply:
Refactored to make the `AerospikeDB` look cleaner

Reply:
Ah ok! I thought Aerospike Go client must have an interface that we can mock but doesn't look like it does so this makes sense. Thanks!